roadmap/implementers-guide/src/types/overseer-protocol.md

```diff
@@ -495,6 +519,13 @@ enum NetworkBridgeMessage {
         /// authority discovery has failed to resolve.
         failed: oneshot::Sender<usize>,
     },
+    /// Inform the distribution subsystems about the new
+    /// gossip network topology formed.
+    NewGossipTopology {
+        /// Ids of our neighbors in the new gossip topology.
+        /// We're not necessarily connected to all of them, but we should.
+        our_neighbors: HashSet<AuthorityDiscoveryId>,
```

Comment:
I would like some discussion in the guide about the expected behavior when onboarding new neighbors in the gossip subsystems. Specifically about the mocked view update we do. On this variant I think would be sufficient, but at some point we should write a high-level section about the network protocol methodology & design.

Reply:
OK, I now see you added that in `gossip-support`, but I think it would be more likely found by gossip protocol authors here.

Reply:
Good point, I'll move it here.